include/opcode/riscv-opc.h

```diff
@@ -934,6 +934,52 @@
 #define MASK_SLET     0xFE00707F
 #define MASK_ADDN     0xC000707F
 
+/* Loads */
+/* Register-immediate with post-increment */
+#define MASK_LPOST      0x707f
+
+#define MATCH_LBPOST    0x000b
+#define MATCH_LBUPOST   0x400b
+#define MATCH_LHPOST    0x100b
+#define MATCH_LHUPOST   0x500b
+#define MATCH_LWPOST    0x200b
+
+/* Register-register with post-increment */
+#define MASK_LRRPOST    0xfe00707f
+
+#define MATCH_LBRRPOST  0x0000700b
+#define MATCH_LBURRPOST 0x4000700b
+#define MATCH_LHRRPOST  0x1000700b
+#define MATCH_LHURRPOST 0x5000700b
+#define MATCH_LWRRPOST  0x2000700b
+
+/* Register-register */
+#define MASK_LRR        0xfe00707f
+
+#define MATCH_LBRR      0x00007003
+#define MATCH_LBURR     0x40007003
+#define MATCH_LHRR      0x10007003
+#define MATCH_LHURR     0x50007003
+#define MATCH_LWRR      0x20007003
+
+/* Post-Incrementing stores */
+#define MASK_SPOST     0x707f
+#define MATCH_SBPOST   0x002b
+#define MATCH_SHPOST   0x102b
+#define MATCH_SWPOST   0x202b
+
+/* Reg-Reg Post-Incrementing stores */
+#define MASK_SRRPOST   0xfe00707f
+#define MATCH_SBRRPOST 0x0000402b
+#define MATCH_SHRRPOST 0x0000502b
+#define MATCH_SWRRPOST 0x0000602b
+
+/* Reg-Reg stores */
+#define MASK_SRR       0xfe00707f
+#define MATCH_SBRR     0x00004023
+#define MATCH_SHRR     0x00005023
+#define MATCH_SWRR     0x00006023
+
```

Comment:
Please make the commenting here a little more consistent i.e. For load instructions there is a heading comment, however, for stores there is a comment on each sub-section mentioning they are store instructions.